Bluetooth: Mesh: Add all missing configuration client model shell command (#40109)
zephyrproject-rtos/zephyr: doc/reference/bluetooth/mesh/shell.rst

	* ``Model ID``: The model ID of the model to add the subscription to.
	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.

``mesh mod-sub-ow <elem addr> <sub addr> <Model ID> [Company ID]``
-------------------------------------------------------------------

> 	Overwrite a model from a group address.

Reviewer: ```suggestion
	Overwrite all model subscriptions with a single new group address.
```

	* ``elem addr``: Address of the element the model is on.
	* ``sub addr``: 16-bit group address the model should added to the subscription list (``0xc000`` to ``0xFEFF``).
	* ``Model ID``: The model ID of the model to add the subscription to.
	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.

``mesh mod-sub-ow-va <elem addr> <Label UUID> <Model ID> [Company ID]``
------------------------------------------------------------------------

> 	Overwrite the model to a virtual address. Models only receive messages sent to their unicast address or a group or virtual address they subscribe to. Models may subscribe to multiple group and virtual addresses.

Reviewer: ```suggestion
	Overwrite all model subscriptions with a single new virtual address. Models only receive messages sent to their unicast address or a group or virtual address they subscribe to. Models may subscribe to multiple group and virtual addresses.
```

The rest of this line should be included in `mod-sub-ow` as well, or not at all

	* ``elem addr``: Address of the element the model is on.
	* ``Label UUID``: 128-bit label UUID of the virtual address as the new Address to be added to the subscription list. Any omitted bytes will be zero.
	* ``Model ID``: The model ID of the model to add the subscription to.
	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.

``mesh mod-sub-del-all <elem addr> <Model ID> [Company ID]``
-------------------------------------------------------------------

> 	Unsubscribe all group address of a model.

Reviewer: ```suggestion
	Remove all group and virtual address subscriptions from a model.
```

	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.

``mesh krp <NetKeyIdx> [Transition]``
-------------------------------------

> 	Get or set the krp value.

Reviewer: ```suggestion
	Get or set the key refresh phase of a subnet.
```

	* ``NetKeyIdx``: The identified network key used to Get/Set the current Key Refresh Phase state.

> 	* ``Transition``: New Key Refresh Phase Transition.

Reviewer: `phase` instead?

What are the valid values for this?

Author: 4.3.2.59 Config Key Refresh Phase Set

Reviewer: Yes, perhaps it could be listed in the docs?

Author: How to do ?

Reviewer: ```suggestion
	* ``Phase``: New Key Refresh Phase. Valid phases are 0, 1 or 2.
```
